content/en/docs/deployment/hub/hub.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 'IoT Hub'
+description: 'Running hub on Kubernetes'
+docsOthersDisplay: true
+date: '2021-11-01'
+categories: [deployment, kubernetes]
+keywords: [deployment, kubernetes, k8s, helm, chart]
+weight: 10
+isMainSectionPage: true
+---
+
+![Deployment diagram](/docs/deployment/hub/static/hub-deployment-diagram.drawio.svg "medium-zoom-image")
+
+Quickest way how to install your own instance of the plgd hub is to use the Mock OAuth2.0 Server. With this option enabled, no authentication is available. The JWT token is automatically issued for the default user.
+
+{{< warning >}}
+The Mock OAuth Server should only be utilized for **testing** and **development purposes**. Exercise additional caution when using it! For production environments, please follow the deployment instructions provided in the [Advanced Configuration](/docs/deployment/hub/advanced) section to ensure proper setup.
+{{< /warning >}}
+
+```sh
+echo "
+global:
+  domain: \"example.com\"
+  hubId: \"d03a1bb4-0a77-428c-b78c-1c46efe6a38e\"
+mockoauthserver:
+  enabled: true" > withMock.yaml
+
+helm upgrade -i -n plgd --create-namespace -f withMock.yaml hub plgd/plgd-hub
+```
+
+{{< note >}}
+
+To use `microk8s` or a similar kubernetes system, ensure that the `ingress`, `dns`, and `storage` addons are enabled. Additionally, configure your Kubernetes cluster to resolve the private domain `*.example.com` through the designated DNS server. For microk8s, use [microk8s enable dns:{DNS_SERVER}](https://microk8s.io/docs/addon-dns) to set up the DNS server.
```

Comment:
"ensure that the ingress, dns, and storage addons are enabled" should be "ensure that the ingress, dns, and storage add-ons are enabled."

Reply:
fixed